decidim-core/app/models/decidim/scope.rb

```diff
@@ -76,6 +76,11 @@ def part_of_scopes(root = nil)
     # Allow ransacker to search for a key in a hstore column (`name`.`en`)
     ransacker_i18n :name
 
+    def self.ransackable_attributes(_auth_object = nil)
+      # %w(code created_at decidim_organization_id id name parent_id part_of scope_type_id updated_at)
```

Comment:
```suggestion
```

Reply:
I'm just leaving these two, but we should remove the rest if they're not necessary